native/src/main/java/io/ballerina/stdlib/graphql/runtime/schema/types/SchemaType.java

```diff
@@ -93,6 +99,10 @@ public void addEnumValue(Object o) {
         this.enumValues.add(o);
     }
 
+    public void addInputField(InputValue inputVal) {
```

Comment:
This is wrong! Please read the spec carefully. A Type cannot have any inputs. Fields can have inputs. The `SchemaField` class already has `addArg()` and `getArgs()` function. Make sure that you're implementing the spec correctly.